safe-global/safe-wallet-web
# [TX flow] Replace tx

## What it solves

Resolves #2058 

## How this PR fixes it

- Moves ReplaceTx to the new flow
- Adds an illustration
- Fixes zIndex issues for the modal

## How to test it

1. Open a Safe
2. Replace an existing transaction
3. Observe the screen in light and dark mode
4. Open the safe list sidebar
5. Observe no overlaps
6. Switch to mobile view
7. Observe no overlaps in the modal

## Screenshots
<img width="750" alt="Screenshot 2023-06-26 at 16 07 43" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/7147f011-1797-4c74-bb93-b337e06febd3">
<img width="1216" alt="Screenshot 2023-06-26 at 16 07 50" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/1dfed183-87eb-4b9f-a24b-66c89745ab6e">
<img width="523" alt="Screenshot 2023-06-26 at 16 07 59" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/a72e00bb-32ad-421e-a7e7-41c359d3cf8e">


## Checklist
